package.json

```diff
@@ -36,87 +79,41 @@
       "git add"
     ]
   },
-  "dependencies": {
-    "@types/crypto-js": "^3.1.43",
-    "@types/express": "^4.16.1",
-    "@types/js-cookie": "^2.2.0",
-    "@types/node": "^10.12.18",
-    "@types/vue": "^2.0.0",
-    "awesome-typescript-loader": "^5.2.1",
-    "c3": "^0.4.20",
-    "cross-env": "^5.2.0",
-    "crypto-js": "^3.1.9-1",
-    "express": "^4.16.4",
-    "js-cookie": "^2.2.0",
-    "moment": "^2.22.1",
-    "natural-orderby": "^2.0.1",
-    "polyfill-array-includes": "^2.0.0",
-    "serve-static": "^1.13.2",
-    "source-map-loader": "^0.2.4",
-    "stylus": "^0.54.5",
-    "stylus-loader": "^3.0.1",
-    "tapable": "^1.1.1",
-    "ts-loader": "^5.3.3",
-    "vue-resource": "^1.4.0",
-    "vue-router": "^3.0.1",
-    "vue-types": "^1.3.4",
-    "vuejs-datepicker": "^1.5.4",
-    "webpack-cli": "^3.2.1"
-  },
-  "devDependencies": {
-    "@commitlint/cli": "^7.2.1",
-    "@commitlint/config-conventional": "^7.1.1",
-    "@osu-cass/tslint-config": "^2.0.0",
-    "autoprefixer": "^7.1.2",
-    "babel-core": "^6.22.1",
-    "babel-helper-vue-jsx-merge-props": "^2.0.3",
-    "babel-loader": "^7.1.5",
-    "babel-plugin-syntax-jsx": "^6.18.0",
-    "babel-plugin-transform-runtime": "^6.22.0",
-    "babel-plugin-transform-vue-jsx": "^3.5.0",
-    "babel-preset-env": "^1.3.2",
-    "babel-preset-stage-2": "^6.22.0",
-    "chalk": "^2.0.1",
-    "copy-webpack-plugin": "^4.0.1",
-    "css-loader": "^2.1.1",
-    "extract-text-webpack-plugin": "^4.0.0-beta.0",
-    "file-loader": "^1.1.4",
-    "friendly-errors-webpack-plugin": "^1.6.1",
-    "html-webpack-plugin": "^3.2.0",
-    "husky": "^1.3.1",
-    "lint-staged": "^8.1.0",
-    "node-notifier": "^5.1.2",
-    "optimize-css-assets-webpack-plugin": "^5.0.1",
-    "ora": "^1.2.0",
-    "portfinder": "^1.0.13",
-    "postcss-import": "^11.0.0",
-    "postcss-loader": "^2.0.8",
-    "postcss-url": "^7.2.1",
-    "prettier": "^1.16.1",
-    "rimraf": "^2.6.0",
-    "semver": "^5.3.0",
-    "shelljs": "^0.7.6",
-    "tslint": "^5.12.1",
-    "typescript": "^3.2.2",
-    "uglifyjs-webpack-plugin": "^1.1.1",
-    "url-loader": "^1.1.2",
-    "vue": "^2.5.22",
-    "vue-loader": "^15.5.1",
-    "vue-style-loader": "^3.0.1",
-    "vue-template-compiler": "^2.5.2",
-    "vue-template-loader": "^1.0.0",
-    "webpack": "^4.28.4",
-    "webpack-bundle-analyzer": "^2.9.0",
-    "webpack-dev-server": "^3.1.14",
-    "webpack-merge": "^4.1.0"
-  },
-  "engines": {
-    "node": ">= 6.0.0",
-    "npm": ">= 3.0.0"
-  },
-  "browserslist": [
-    "> 1%",
-    "last 2 versions",
-    "not ie <= 8"
-  ]
+  "jest": {
```

Comment:
Could we move this into the `jest.config.js`?

Reply:
I prefer all config that is able to be in the `package.json` to be there. Since the `test` npm script is also there, we only need to go to one place to configure jest.